addons/website_slides/static/src/js/public/components/slide_quiz_finish_dialog/slide_xp_progress_bar.xml

```diff
@@ -2,12 +2,7 @@
 <templates xml:space="preserve">
```

Comment:
Would you mind linking the commit that upgrades to bootstrap 5.3 in the commit message?